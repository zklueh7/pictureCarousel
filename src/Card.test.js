import { render, fireEvent } from "@testing-library/react";
import Card from "./Card";
// import TEST_CARDS from "./_testCommon.js";

it("renders without crashing", function() {
  render(<Card
    caption="captionnn"
    src="srcc" 
    currNum={0}
    totalNum={3}
  />)
});

it("matches snapshot", function() {
    const {asFragment} = render(<Card
        caption="captionnn"
        src="srcc" 
        currNum={0}
        totalNum={3}
      />);
      
      expect(asFragment()).toMatchSnapshot();
});