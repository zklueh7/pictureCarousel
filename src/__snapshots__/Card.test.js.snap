// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    >
      captionnn
    </h4>
    <img
      alt="captionnn"
      class="Card-image"
      src="srcc"
    />
    <small
      class="Card-small"
    >
      Image 0 of 3.
    </small>
  </div>
</DocumentFragment>
`;
